hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/benefitInformation.jsp

```diff
@@ -24,27 +24,33 @@
 
 <div id="${n}dl-benefit-summary" class="fl-widget portlet dl-benefit-summary hrs">
   <div class="dl-banner-links">
-    <c:choose>
-    <c:when test="${enrollmentFlag == 'O'}">
+    <sec:authorize ifAnyGranted="ROLE_VIEW_BENEFITS">
       <div class="dl-banner-link">
-        <spring:message code="benefit.summary.enrollment.O.message" text="You have a benefit enrollment opportunity. Please enroll online by clicking the following link: "/>
+        You have a benefit enrollment opportunity.
         <a target="_blank" href="${hrsUrls['Open Enrollment/Hire Event']}"><spring:message code="open.enrollment" text="Open Enrollment" /></a>
         <c:if test="${isMadisonUser}">
-          ${learnMoreEBenefitGuide}
+            ${madisonNewLearnMore}
```

Comment:
I suspect you didn't mean a bare URL to be here, but I also expect that'll take care of itself when those values are inlined into the JSP and it becomes more obvious what they are and how they relate to the rest of the markup.